src/directives/DraggableWin.js

```diff
@@ -108,11 +107,18 @@ export const DraggableWin = {
       // id="drag-header" or class="drag-header" should be draggable.
       // If the user clicked another area, do nothing.
       // By default it is restricted to class="wgu-win-title"
+
+      // get the element path
+      const path = e.path || (e.composedPath && e.composedPath());
+
       if (_data.draggableElementSelector &&
-          !checkIfSelectorInPath(_data.draggableElementSelector, e.path)) {
+          !checkIfSelectorInPath(_data.draggableElementSelector, path)) {
         return;
       }
 
+      // set the offset for each moousedown
```

Comment:
```suggestion
      // set the offset for each mousedown
```

Reply:
Hi @JakobMiksch,

To reproduce the issue, open two windows configured with `draggable: true` and drag one window after the other

**Before this PR**, the current window being dragged will always offset to the position of the previously dragged window.

**After this PR**, the offset is resetted on each drag start.

This issue is caused by the following object being shared across all components that use the `DraggableWin.js` directive.
https://github.com/wegue-oss/wegue/blob/7f53a55545015d37c5b4fc73c8f9dba334240b7d/src/directives/DraggableWin.js#L9-L26

Reply:
Alright. I have actually never used this dragging functionality so far. 
I only could drag the InfoClick window so far. The other ones do not work to drag. 
Do you know why, which windows do you use for testing?

Reply:
I tested with `wgu-maprecorder` and the `wgu-infoclick` in google chrome. Had to set `draggable: true` to both of them in the app-conf.json.  Are you able to reproduce?

Reply:
Yes, I was able to reproduce. Good catch.

Reply:
Other windows like the measure window do not get draggable even with the `draggable: true` in the `app-conf.json`. Reason is the positioning in the projects CSS file, where already a TODO is noted, that the positioning has to be harmonized, see https://github.com/wegue-oss/wegue/blob/master/src/assets/css/wegue.css#L112